src/coupling.jl

```diff
@@ -49,8 +51,10 @@ function cat_problems(prob::DiscreteProblem, prob_control::DiffEqBase.AbstractOD
     ODEProblem(new_f, u0_coupled, prob.tspan, prob.p)
 end
 
-function cat_problems(prob::DiffEqBase.AbstractSDEProblem,
-                      prob_control::DiffEqBase.AbstractSDEProblem)
+function cat_problems(
+        prob::DiffEqBase.AbstractSDEProblem,
+        prob_control::DiffEqBase.AbstractSDEProblem,
+)
```

Comment:
This seems to be putting the `)` on a new line and leaving it unindented when the discussion in that long issue was that it should just state at the end of the last argument and sub-lines of the function signature should be double indented always.

Reply:
solved in most recent commit